src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/high-risk-data-breaches/View.tsx

```diff
@@ -0,0 +1,39 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// "use client";
+
+import { GuidedExperienceBreaches } from "../../../../../../../functions/server/getUserBreaches";
+import Link from "next/link";
+import { HighRiskBreachLayout } from "./HighRiskBreachLayout";
+
+type Props = {
+  breaches: GuidedExperienceBreaches;
+};
+export const View = (props: Props) => {
+  return (
+    <div>
+      <HighRiskBreachLayout
+        typeOfBreach="creditCard"
+        breachData={props.breaches}
+      />
+
+      <Link href="/redesign/user/dashboard/fix/high-risk-data-breaches/social-security-number">
+        SSN Breaches
+      </Link>
+      <br />
+      <Link href="/redesign/user/dashboard/fix/high-risk-data-breaches/credit-card-number">
+        Credit card
+      </Link>
+      <br />
+      <Link href="/redesign/user/dashboard/fix/high-risk-data-breaches/bank-account">
+        Bank Account
+      </Link>
+      <br />
+      <Link href="/redesign/user/dashboard/fix/high-risk-data-breaches/pin-number">
+        Pin Number
```

Comment:
nit: not sure if this should just be "PIN".

Reply:
I use my PIN number to get my USD Dollars from the ATM Machine.

(But this is just a temporary link for developers I think (hence also not localised), so it shouldn't matter. Also, I think that ship has sadly sailed anyway...)